src/sound/soundbase.cpp

```diff
@@ -232,98 +221,142 @@ QVector<QString> CSoundBase::LoadAndInitializeFirstValidDriver ( const bool bOpe
 }
 
 /******************************************************************************\
-* MIDI handling                                                                *
+* Command Line Handling                                                        *
 \******************************************************************************/
 void CSoundBase::ParseCommandLineArgument ( const QString& strMIDISetup )
 {
-    int iMIDIOffsetFader = 70; // Behringer X-TOUCH: offset of 0x46
-
-    // parse the server info string according to definition: there is
-    // the legacy definition with just one or two numbers that only
-    // provides a definition for the controller offset of the level
-    // controllers (default 70 for the sake of Behringer X-Touch)
-    // [MIDI channel];[offset for level]
-    //
-    // The more verbose new form is a sequence of offsets for various
-    // controllers: at the current point, 'f', 'p', 's', and 'm' are
-    // parsed for fader, pan, solo, mute controllers respectively.
-    // However, at the current point of time only 'f' and 'p'
-    // controllers are actually implemented.  The syntax for a Korg
-    // nanoKONTROL2 with 8 fader controllers starting at offset 0 and
-    // 8 pan controllers starting at offset 16 would be
-    //
-    // [MIDI channel];f0*8;p16*8
-    //
-    // Namely a sequence of letters indicating the kind of controller,
-    // followed by the offset of the first such controller, followed
-    // by * and a count for number of controllers (if more than 1)
-    if ( !strMIDISetup.isEmpty() )
+    if ( strMIDISetup.isEmpty() )
```

Comment:
I'd prefer curly braces...
The code in general doesn't - I know

Reply:
Existing code but I agree.  However, the style guide says it's accepted.